spring-cloud-generator/generate-one.sh

```diff
@@ -17,6 +19,7 @@ do
         i) client_lib_artifactid=${OPTARG};;
```

Comment:
Cannot comment on unchanged code, but L18 `v) version=${OPTARG};;` and it's usage(shown below) are probably not needed anymore. Can you double check and remove them?
```
sed -i 's/{{client-library-version}}/'"$version"'/' "$starter_artifactid"/pom.xml
```

Reply:
Fixed in https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/1430